Makefile.docker

```diff
@@ -26,6 +27,9 @@ docker_tag:
 	echo "Tagging strimzi/$(PROJECT_NAME):$(DOCKER_TAG) to $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG) ..."
 	docker tag strimzi/$(PROJECT_NAME):$(DOCKER_TAG) $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG)
 
-docker_push: docker_tag
-	echo "Pushing $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG) ..."
-	docker push $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG)
\ No newline at end of file
+docker_push:
+	echo "Building and pushing $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG) ..."
+	mkdir -p scripts
+	cp ../../scripts/run.sh ./scripts/run.sh
+	docker buildx build --push --platform=$(PLATFORMS) --build-arg version=$(DOCKER_VERSION_ARG) --tag $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(PROJECT_NAME):$(DOCKER_TAG) $(DOCKERFILE_DIR)
+	rm -rf ./scripts
```

Comment:
I didn't work with `docker buildx` command, but do we need to have build/tag/push in one command? Because if we do, goals like `docker_build` and `docker_tag` are not necessary and `docker_push` should be renamed, right? If not, why we cannot build it in `docker_build`, tag it in `docker_tag` and push it in `docker_push`?

Reply:
Hi @im-konge , thanks for the comment.

Yes,  `docker buildx` will complete build/tag/push in one command. Thus `docker_push` should be renamed to `docker_build_tag_push` and `docker_tag` is not necessary, but I think we could keep `docker_build` target to build amd64 images for local usage.

As mentioned above, I will follow the same logic adopted in other Strimzi repos so that we can separate tasks in `docker_build`, `docker_tag` and `docker_push` goals.